### PR TITLE
Flake8

### PR DESCRIPTION
**Pull request by benvand**
Fix Flake8 violations as per attached file
[flake8_violations.txt](https://github.com/alphagov/digitalmarketplace-buyer-frontend/files/1122318/flake8_violations.txt)

### CODE REVIEW

File: app/main/errors.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def page_not_found(e):
 
 
 @main.app_errorhandler(410)
-def page_not_found(e):
```
Comment (benvand) [inline app/main/errors.py]: 410 is `gone` not `not_found` this was a copy-paste error

File: tests/main/presenters/test_service_presenters.py [hunk 2]
```diff
@@ -122,12 +122,6 @@ def test_external_framework_url_returns_correct_suffix(self):
         assert self.meta.get_external_framework_url({'frameworkSlug': 'None'}) is None
 
     def test_get_documents_returns_the_correct_document_information(self):
-        keys = [
```
Comment (benvand) [inline tests/main/presenters/test_service_presenters.py]: Unused

File: tests/main/views/test_marketplace.py [hunk 3]
```diff
@@ -336,7 +336,7 @@ def test_cookie_page(self):
         assert res.status_code == 200
         assert '<h1>Cookies</h1>' in self._strip_whitespace(res.get_data(as_text=True))
 
-    def test_cookie_page(self):
```
Comment (benvand) [inline tests/main/views/test_marketplace.py]: Was redefining the test above, copy-paste error

File: tests/main/views/test_marketplace.py [hunk 4]
```diff
@@ -359,28 +359,24 @@ def teardown_method(self, method):
 
 class TestBriefPage(BaseBriefPageTest):
```
Comment (benvand) [inline tests/main/views/test_marketplace.py]: This method was only used once so 'intrapolated' back into the test and actually perform assertions

File: tests/main/views/test_marketplace.py [hunk 5]
```diff
@@ -769,6 +765,11 @@ def test_dateformat_in_withdrawn_banner_displayed_correctly(self):
 
 
 class TestCatalogueOfBriefsPage(BaseApplicationTest):
```
Comment (benvand) [inline tests/main/views/test_marketplace.py]: DRY static helper method

File: tests/main/views/test_marketplace.py [hunk 6]
```diff
@@ -914,10 +915,11 @@ def test_catalogue_of_briefs_page_filtered(self):
         parsed_next_url = urlparse(document.xpath("//li[@class='next']/a/@href")[0])
         assert parsed_original_url.path == parsed_prev_url.path == parsed_next_url.path
```
Comment (benvand) [inline tests/main/views/test_marketplace.py]: Prefer brackets over line continuation characters

File: tests/main/views/test_services.py [hunk 7]
```diff
@@ -209,14 +209,14 @@ def test_enabled_service_has_unavailable_banner(self):
...
-                service=self.service['services']
-            )
+        audit_event = self._get_status_update_audit_event_for(
+            update_type='update_service_status',
+            old_status='published',
+            new_status='enabled',
+            timestamp='2016-01-05T17:01:07.649587Z',
+            service=self.service['services']
+        )
+        self.service['serviceMadeUnavailableAuditEvent'] = audit_event
```
Comment (samuelhwilliams) [inline tests/main/views/test_services.py]: What rule forces this intermediate step? Seems odd.

Comment (TheDoubleK) [inline tests/main/views/test_services.py]: Yep, getting rid of the line continuation is nice, but feels like the assignment can still all be done in one step.

File: tests/main/views/test_services.py [hunk 8]
```diff
@@ -237,14 +237,14 @@ def test_enabled_service_has_unavailable_banner(self):
...
-                service=self.service['services']
-            )
+        audit_event = self._get_status_update_audit_event_for(
+            update_type='update_service_status',
+            old_status='published',
+            new_status='disabled',
+            timestamp='2016-01-05T17:01:07.649587Z',
+            service=self.service['services']
+        )
+        self.service['serviceMadeUnavailableAuditEvent'] = audit_event
```
Comment (TheDoubleK) [inline tests/main/views/test_services.py]: Same here.

File: tests/main/views/test_services.py [hunk 9]
```diff
@@ -265,14 +265,15 @@ def test_disabled_service_has_unavailable_banner(self):
...
-                service=self.service['services']
-            )
+        audit_event = self._get_status_update_audit_event_for(
+            update_type='framework_update',
+            old_status='live',
+            new_status='expired',
+            timestamp='2016-01-05T17:01:07.649587Z',
+            service=self.service['services']
+        )
+        self.service['serviceMadeUnavailableAuditEvent'] = audit_event
```
Comment (TheDoubleK) [inline tests/main/views/test_services.py]: etc... :)

**Comment by TheDoubleK** [comment]
Generally looks good, I'm with Sam about the unnecessary intermediate assignment though.

File: requirements_for_test.txt [hunk 10]
```diff
@@ -1,14 +1,14 @@
...
 #Required for testing
-pytest==2.9.1
-pep8==1.5.7
-mock==2.0.0
-lxml==3.4.4
-cssselect==0.9.1
-watchdog==0.8.3
-
 coverage==3.7.1
+cssselect==0.9.1
```
Comment (galund) [inline requirements_for_test.txt]: did these all have to move around?

**Comment by galund** [comment]
This looks really nearly ready to go @benvand, can we can we get Sam/Kev's issues sorted, and the conflicts resolved? No need to rebase if a merge will do, if that's any easier, in my opinion.

**Pull request closed without merging**